### PR TITLE
feat: Validation of opportunity feeds

### LINKED ISSUE #89: Run RPDE validation rules
Run RPDE validation rule across each page, to check they're consistent

### PR DESCRIPTION
**Pull request by nickevansuk**
Add inline validation to feeds, which closes #89

### CODE REVIEW

File: packages/openactive-broker-microservice/app.js [hunk 1]
```diff
@@ -62,6 +71,108 @@ const startTime = new Date();
...
+const validationResults = new Map();
+
+/**
+ * Use OpenActive validator to validate the opportunity
+ *
+ * @param {any} data opportunity JSON-LD object
+ */
+async function storeValidationResults(data) {
+  const id = data['@id'] || data.id;
+  const errors = await validateItem(data, false);
```
Comment (civsiv) [inline packages/openactive-broker-microservice/app.js:111]: If `isOrdersFeed` is always false, is it needed? Or are you anticipating validating Orders Feeds too? If so, maybe a comment to explain?

File: packages/openactive-broker-microservice/app.js [hunk 2]
```diff
@@ -62,6 +71,108 @@ const startTime = new Date();
...
+}
+
+const validationResults = new Map();
+
+/**
+ * Use OpenActive validator to validate the opportunity
+ *
+ * @param {any} data opportunity JSON-LD object
+ */
+async function storeValidationResults(data) {
```
Comment (civsiv) [inline packages/openactive-broker-microservice/app.js:109]: This function both validates and stores results, so maybe should be called something like `validateAndStoreValidationResults`?

**Review by civsiv — APPROVED** [review]

File: packages/openactive-broker-microservice/app.js (hunk 1 above)
Comment (nickevansuk) [reply packages/openactive-broker-microservice/app.js:111]: Good point, YAGNI, have removed

File: packages/openactive-broker-microservice/app.js (hunk 2 above)
Comment (nickevansuk) [reply packages/openactive-broker-microservice/app.js:109]: Good shout, have updated

**Pull request merged**